### ISSUE TITLE
As a GM, I want to set character sheets for my players in online mode.

### ISSUE THREAD

**Issue by sqwishy** [fariapp/fari#112]
## 🚩 Is your feature request related to a problem? Please describe

I've prepared player character sheets in Fari for my session. In order for players to use them in an online session, apparently they have to have the character sheets imported into Fari on their client, and use them when connecting.

## ✅ Describe the solution you'd like

If a player joins without a character sheet, the button that opens it, which is disabled in this case, gives me the option to add a character sheet, either by creating a new one or using an existing one -- similar to the dialog from the "Add Character" button in offline.

If they already have a character sheet, there is a button somewhere to replace it that basically does the above. (Maybe a button to remove it?) Or, when you open the character sheet, there are options (next to the save button?) to save it locally or replace it with a character sheet you've already created or imported into Fari.

_(Hopefully this isn't already a thing and I just missed it.)_

**Comment by RPDeshaies** [comment]
So when you create a session you would like to have the ability to select a "character sheet template" and then users clicking on the character sheet icon would be popped up with a character sheet based on the template you assigned to the scene, am I understanding that correctly?

![image](https://user-images.githubusercontent.com/6224111/92778291-04764280-f36f-11ea-89b6-6e81614cb616.png)

**Comment by player-03** [comment]
I support any and all of the ideas in this thread. I've been introducing new players to Fari, and it'd be great to have some way - any way - to give them a sheet after they inevitably log in without one.

Bonus points for being able to swap sheets mid-session. That sounds great too.

**Comment by tgirod** [comment]
I agree with @sqwishy, beeing able to assign a character sheet to a player would be great. Right now I'm prepping a game and transcribing my players paper character sheet into fari to make the transition smoother. Right now my solution is this:

1. create the character sheet
2. export it as json files
3. send the file to my player
4. tell my player to go to fari.app and import the file (a player cannot import anything from a game link, if I'm not mistaken)
5. send them the link to the game
6. ask them to join the game with the imported character

This could be smoother. For example, just beeing able to import a character sheet on the "join game" page would be a massive improvement.

**Comment by RPDeshaies** [comment]
I totally forgot to update this issue but I wanted to let you all know that this is possible with the latest version of fari available at https://fari.app/ 

Thanks for all your feedback and hope you like the new feature 🥳
